subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -207,52 +183,68 @@ static int fem_nrf21540_gpio_configure(void)
 		},
 		.pa_pin_config = {
 #if DT_NODE_HAS_PROP(DT_NODELABEL(nrf_radio_fem), tx_en_gpios)
-			.enable       = true,
-			.active_high  =
+			.cfg_type      = MPSL_FEM_PIN_CFG_TYPE_PIN_PORT,
+			.enable        = true,
+			.active_high   =
```

Comment:
nit: it should be possible now to unwrap the lines
```c
.active_high   = MPSL_FEM_GPIO_POLARITY_GET(tx_en_gpios),
.p_port        = MPSL_FEM_GPIO_PORT_REG(tx_en_gpios),
.gpio_port_no  = MPSL_FEM_GPIO_PORT_NO(tx_en_gpios),
.gpio_port_pin = MPSL_FEM_GPIO_PIN_NO(tx_en_gpios),
```
here and in other occurences as well.

Reply:
Thank you for addressing the comment :)